# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xorg/imake_rules.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./config/cf/Imake.rules.orig	2003-03-23 11:44:47.000000000 +0100
+++ ./config/cf/Imake.rules	2003-03-23 11:50:43.000000000 +0100
@@ -1693,8 +1693,9 @@
 		(cd Concat($(DESTDIR),npath/mdir); \			@@\
 		 tar xf -; exit 0); exit 0); \				@@\
 	  fi; \								@@\
-	  $(RM) -r rdir; \						@@\
-	  $(LN) Concat($${RELPATH},npath/mdir) ldir; \			@@\
+	  if [ ! -e rdir ]; then \					@@\
+	    $(LN) Concat($${RELPATH},npath/mdir) ldir; \		@@\
+	  fi; \								@@\
 	fi
 #else
 #define LinkConfDirectoryLong(mdir,cdir,rdir,ldir,opath,npath)
